src/main/java/io/vlingo/wire/fdx/bidirectional/rsocket/RSocketClientChannel.java

```diff
@@ -159,7 +159,7 @@ private void handle(Payload payload) {
         final ByteBuffer payloadData = payload.getData();
         final ConsumerByteBuffer put = pooledBuffer.put(payloadData);
         consumer.consume(put.flip());
-      } catch (final Throwable e) {
+      } catch (final Exception e) {
```

Comment:
Failing to catch all exceptions here would propagate the errors and will might stop the channel from consuming. Also it will not release the `pooledBuffer` on unchecked exceptions.